mitosheet/src/mito/components/endo/ColumnHeaderDropdown.tsx

```diff
@@ -39,188 +38,66 @@ export default function ColumnHeaderDropdown(props: {
         }
     }, [props.display])
         
-
-    // If the user interacts with the column header dropdown, we always default to 
-    // selecting the first row for any action, like setting the column formula. 
-    const rowIndex = 0 
-    const columnIndex = Object.keys(props.sheetData.columnIDsMap).indexOf(props.columnID)
+    const getPropsForDropdownItem = (actionName: ActionEnum) => {
+        const action = props.actions.buildTimeActions[actionName];
+        return {
+            title: action.toolbarTitle ?? action.longTitle,
+            onClick: () => {
+                props.closeOpenEditingPopups();
+                void action.actionFunction();
+            },
+            icon: action.contextMenuIcon ? <action.contextMenuIcon /> : action.toolbarIcon ? <action.toolbarIcon/> : <StepsIcon/>,
+            rightText: action.displayKeyboardShortcuts?.mac
+        }
+    }
 
     return (
         <Dropdown
             display={props.display}
             closeDropdown={() => props.setOpenColumnHeaderDropdown(false)}
-            width='medium'
         >
+            <DropdownItem {...getPropsForDropdownItem(ActionEnum.Copy)} />
+
+            <DropdownSectionSeperator isDropdownSectionSeperator={true}/>
+
             <DropdownItem
+                {...getPropsForDropdownItem(ActionEnum.Add_Column_Left)}
                 title='Insert Column Left'
```

Comment:
I think these titles can be moved to the actions like the icons. So we have `toolbarTitle` and also `contextMenuTitle`. I would actually be down for `titleToolbar` and `titleContextMenu` -- like icons, this makes the org of the actions very nice and easy to parse and maintain.